packages/core/src/browser/about-dialog.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+import { Message } from '@phosphor/messaging';
+import { AbstractDialog } from './dialogs';
+import { ExtensionInfo, ApplicationInfo } from '../common/application-protocol';
+
+export const ABOUT_CONTENT_CLASS = 'theia-aboutDialog';
+export const ABOUT_EXTENSIONS_CLASS = 'theia-aboutExtensions';
+
+export class AboutDialog extends AbstractDialog<void> {
+
+    protected readonly okButton: HTMLButtonElement;
+
+    constructor(extensionsInfos: ExtensionInfo[], applicationInfo?: ApplicationInfo) {
+        super({
+            title: `About Theia`
```

Comment:
Being able to customize it would be nice. At least, please move this to a protected method. For instance, in the Yangster application, it would be great to show `About Yangster`.

Reply:
I don't think it is possible in the super call.

I wonder whether we should put it `package.json` that app developers could brand their apps declaratively, like `applicationName` and fall back to `Theia` if it is not provided. cc @svenefftinge